webservices/src/main/java/com/javaschool/webservices/service/impl/PackageServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.javaschool.webservices.service.impl;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+
+import com.javaschool.webservices.model.PackageType;
+import com.javaschool.webservices.service.PackageService;
+import com.javaschool.webservices.service.ShippingService;
+
+@Service
+public class PackageServiceImpl implements PackageService {
+	
+	private final ShippingService shippingService;
+	
+	public PackageServiceImpl(ShippingService shippingProducerService) {
+		this.shippingService = shippingProducerService;
+	}
+
+	@Override
+	public List<String> getSizes() {
+		List<String> sizesList = new LinkedList<>();
```

Comment:
Why did you choose to use a `LinkedList` here instead of an `ArrayList` ?

Reply:
It was only for test purpose, i wont forget to add a TODO comment.